lib/src/platform/src/streams_channel.dart

```diff
@@ -24,20 +24,25 @@ import 'package:ably_flutter/ably_flutter.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
+/// @nodoc
 /// Manages multiple event listeners which would otherwise require verbose code
 /// on platform side
 class StreamsChannel {
+  /// @nodoc
   /// initializes with event channel [name] and method [codec]
```

Comment:
All comments in this file could be "upgraded" to sentences - starting with a capital letter and ending with the dot if we want to

Reply:
Done in 19b0814c30acaf767beee3ed6eecd362b672f012